python/shark_turbine/dynamo/importer.py

```diff
@@ -191,6 +185,7 @@ def import_stateless_graph(self, g: Graph, func_name: str = "main"):
             entry_block = Block.create_at_start(func.body, ftype.inputs)
         node_importer = GraphNodeImporter(self._c, self._cc, entry_block)
         node_importer.import_nodes(g.nodes)
+        self.symbol_table.insert(func)
```

Comment:
Note that inserting into the symbol table can change the function's symbol, so you need to make sure that anywhere you were using the original name doesn't escape.

Reply:
(this looks fine to me, just noting)